README.md

```diff
@@ -342,21 +342,35 @@ Class2       0            10
 
 <img src="https://github.com/sepandhaghighi/pycm/raw/master/Otherfiles/plot3.png">
 
-### Online help
-
-`online_help` function is added in `version 1.1` in order to open each statistics definition in web browser
+### ROC curve
 
+`ROCCurve`, added in `version 3.7`, is devised to compute the Receiver operating characteristic (ROC) or simply ROC curve. In ROC curves, the Y axis represents the True Positive Rate, and the X axis represents the False Positive Rate. Thus, the ideal point is located at the top left of the curve, and a larger area under the curve represents better performance. ROC curve is a graphical representation of binary classifiers' performance. In PyCM, `ROCCurve` binarizes the output based on the "One vs. Rest" strategy to provide an extension of ROC for multi-class classifiers. Getting the actual labels vector, the target probability estimates of the positive classes, and the list of ordered labels of classes, this method is able to compute and plot TPR-FPR pairs for different discrimination thresholds and compute the area under the ROC curve. 
 
 ```pycon
+    >>> crv = ROCCurve(actual_vector = np.array([1, 1, 2, 2]), probs = np.array([[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]]), classes=[2, 1])
+    >>> crv.thresholds
+    [0.1, 0.2, 0.35, 0.4, 0.6, 0.65, 0.8, 0.9]
+    >>> auc_trp = crv.area()
+    >>> auc_trp[1]
+    0.75
+    >>> auc_trp[2]
+    0.75
+```	
```

Comment:
Maybe we could add a ROC plot here.

Reply:
You are right!
But we need a real classification sample to show a proper image, it's not possible to show a good PR/ROC curve with a small sample. So I think it would be better to leave it for `Example9` in the next version.